src/script/interpreter.h

```diff
@@ -185,6 +185,12 @@ enum : uint32_t {
     // Validating OP_CHECKSIGFROMSTACK
     SCRIPT_VERIFY_CHECKSIGFROMSTACK = (1U << 31),
 
+    // Making OP_PAIRCOMMIT non-standard
+    SCRIPT_VERIFY_DISCOURAGE_PAIRCOMMIT = (1U << 30),
+
+    // Validating OP_PAIRCOMMIT
+    SCRIPT_VERIFY_PAIRCOMMIT = (1U << 31),
```

Comment:
why do we have multiple ops on the same bit? shouldn't you be able to distinguish between them, or there is no point in only validating a subset of those?

Reply:
It's all LNHANCE, we don't even have these separated as script flags in our repo.